.github/workflows/ci.yml

```diff
@@ -114,14 +114,14 @@ jobs:
           docker run --rm --network "$DOCKER_NETWORK" \
               -e RAILS_ENV -e POSTGRES_HOST -e POSTGRES_PORT -e REDIS_URL \
               "$FULL_DOCKER_IMAGE" \
-              bin/rails db:setup
+              bundle exec rails db:setup
```

Comment:
Spring binstubs are fixed, so the places that benefit from Spring are using `bin` and places that don't are using `bundle exec`.